fixing grammar and adding unified styling (Guide) (#164)
lukso-network/docs: docs/guides/universal-profile/browser-extension/03-interact-with-dapp.md

# Interact with a dApp

:::danger
The UP Browser Extenstion is currently in the **development alpha** version. DO NOT use this in production!
:::

:::note

> MetaMask (or any other ethereum provider) cannot be running simultaneously. It would be best to deactivate the MetaMask extension for the UP Extension to work.

Reviewer: ```suggestion
MetaMask (or any other ethereum provider) cannot be running simultaneously. It is recommended to deactivate the MetaMask extension for the UP Extension to work.
```

const etherProvider = new ethers.providers.Web3Provider(window.ethereum);
```

  </TabItem>
</Tabs>

### 2. Get access to the UP address in use.

A call to `requestAccounts` will open an extension popup to authorize an account.

<Tabs>
  <TabItem value="web3" label="web3">

```js
const accountsRequest: string[] = await web3.eth.requestAccounts();
const accounts: string[] = await web3.eth.getAccounts(); //should also yield the same address
```

  </TabItem>
  <TabItem value="etherjs" label="etherjs">

```js
const accountsRequest: string[] = await etherProvider.send(
  'eth_requestAccounts',
  [],
);
const signer = etherProvider.getSigner();
await signer.getAddress(); //should also yield the same address
```

  </TabItem>
  <TabItem value="raw" label="raw">

```js
const accountsRequest: string[] = await window.ethereum.request({
  method: 'eth_requestAccounts',
  params: [],
});
const accounts: string[] = await window.ethereum.request({
  method: 'eth_accounts',
  params: [],
}); //should also yield the same address
```

  </TabItem>
</Tabs>

## Operations

Each operation will open an extension popup window for a user to confirm the transaction or sign the message.

:::note

> If you don't have funds at your EOA address, the application will use the relayer <br/> service by default to pay the gas fees (thus ignoring gas properties).

Reviewer: at? Why not "on" ?

Not sure if the `<br/>` is necessary here? Why add line break here?

Reviewer: `<br>` > `<br/>`

Author: Guess this was a plain Grammarly issue thinking about an IRL address. The line break was just for the looks, but you´re right: Its not necessary here because of the added :::note:::
